tests/test_modeling_tf_transfo_xl.py

```diff
@@ -92,30 +92,21 @@ def create_and_check_transfo_xl_model(self, config, input_ids_1, input_ids_2, lm
         model = TFTransfoXLModel(config)
 
         hidden_states_1, mems_1 = model(input_ids_1).to_tuple()
+        mems_1 = [mem.numpy() for mem in mems_1]
```

Comment:
Same for those here, I don't think you need that line and the equivalent for mems2